src/Exceptions/HttpBadGateway.php

```diff
@@ -8,4 +8,3 @@ class HttpBadGateway extends HttpException {
     public const DESCRIPTION = "Indicates that the server, while acting as a gateway or proxy, received an invalid response from an inbound server it accessed while attempting to fulfill the request.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/502";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces